### PR TITLE
Fix cloudwatchevents typo

### PR DESCRIPTION
**Pull request by sethvincent**
**Summary:** Small PR to address typo

## Changes

* Fix typo

### CODE REVIEW

**Comment by yjpa7145** [comment]
Are there any tests for this method?  Would a simple "happy path" test have caught this?  If so, can you please add one?

**Comment by mennovandiermen** [comment]
Should there be some testing framework for the API that has a run at every endpoint's methods? I realize that may be a really big story/multi-sprint endeavor, so stop me if it sounds crazy.

**Comment by sethvincent** [comment]
@menno I think you're sounding reasonable :smile:. That seems like something we'll go over at the upcoming planning meeting.

File: packages/ingest/aws.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ function sqs(local) {
 
 class Events {
   static async putEvent(name, schedule, state, description = null, role = null) {
-    const cwevents = new aws.cloudwatchevents();
+    const cwevents = new AWS.CloudWatchEvents();
```
Comment (abarciauskas-bgse) [inline packages/ingest/aws.js]: I think the typo is actually with `new` - e.g. the intention was:
```javascript
const cwevents = aws.cloudwatchevents();
```
I think I made this updates for tests

File: .eslint-ratchet-high-water-mark [hunk 2]
```diff
@@ -1 +1 @@
-1874
+1763
```
Comment (abarciauskas-bgse) [inline .eslint-ratchet-high-water-mark]: Perhaps we should revert this change? CI is complaining:
```
eslint-ratchet error: eslint errors have increased from 1763 to 1873
```

File: packages/ingest/aws.js (hunk 1 above)
Comment (mennovandiermen) [reply packages/ingest/aws.js]: Should that change then not also apply to the `deleteEvent`, `putTarget` and `deleteTarget` functions?

Comment (abarciauskas-bgse) [reply packages/ingest/aws.js]: I think the revert is okay for now - I think whatever tests I was writing ended up not getting rolled into the test suite (as CI seems to be passing with this change) - thanks for checking though!

**Comment by jennyhliu** [comment]
When I try to update a rule (from dashboard or via api), I got this:
Error: Could not process scheduled_ingest, Error: : aws.cloudwatchevents is not a constructor
After I update aws.cloudwatchevents() to AWS.CloudWatchEvents(), this error is not there.

**Comment by sethvincent** [comment]
closing this as the issue was resolved in 1.2.0

**Pull request closed without merging**